composer.json

```diff
@@ -53,5 +53,11 @@
     },
     "config": {
         "preferred-install": "dist"
-    }
+    },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "ssh://git@github.com/SocialiteProviders/Microsoft-Graph.git"
```

Comment:
Error with composer setup: users need to have access rights to this repository in order to perform `composer update` function. Maybe you can package a stable version of `socialiteproviders/microsoft-graph` so it can be included in DirectDemocracy's `composer.json` with minimum stability "stable".